spec/requests/api/issues_spec.rb

```diff
@@ -193,6 +193,13 @@
       json_response['iid'].should == issue.iid
     end
 
+    it "should return a project issue by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.